src/current/v23.1/connection-pooling.md

```diff
@@ -25,37 +25,149 @@ Each time an application reads or writes data, it will request one of the connec
 
 {% include {{page.version.version}}/sql/server-side-connection-limit.md %} This may be useful in addition to your connection pool settings.
 
-## Validating connections in a pool
+</section>
 
-After a connection pool initializes connections to CockroachDB clusters, those connections can occasionally break. This could be due to changes in the cluster topography, or rolling upgrades and restarts, or network disruptions. CockroachDB {{ site.data.products.cloud }} clusters periodically are restarted for patch version updates, for example, so previously established connections would be invalid after the restart.
+Cockroach Labs strongly recommends that your application instance run in the same region as your CockroachDB cluster.
 
-Validating connections is typically handled automatically by the connection pool. For example, in HikariCP the connection is validated whenever you request a connection from the pool, and the `keepaliveTime` property allows you to configure an interval to periodically check if the connections in the pool are valid. Whatever connection pool you use, make sure connection validation is enabled when running your application.
+## Monitor SQL connections
+
+To validate that your connection pool is correctly configured, monitor the SQL connection metrics and graphs on your cluster.
+
+### Monitor new connections
+
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+The [`sql.new_conns` metric]({% link {{ page.version.version }}/metrics.md %}#available-metrics) and [SQL Connection Rate graph]({% link {{ page.version.version }}/ui-sql-dashboard.md %}#sql-connection-rate) expose the number of new SQL connections per second.
+</section>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+The [SQL Connection Attempts graph]({% link cockroachcloud/metrics-page.md %}#identify-sql-problems) shows the number of new SQL connections per second.
+</section>
+
+A misconfigured connection pool will result in a very large number of new connections per second.
+
+### Monitor active connections
+
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+
+The [`sql.conns` metric]({% link {{ page.version.version }}/metrics.md %}#available-metrics) and [Active SQL Statements graph]({% link {{ page.version.version }}/ui-sql-dashboard.md %}#active-sql-statements) show the number of active connections per second on your cluster or node.
+
+Using the following formula:
+
+**connections = (number of cores * 4)**
+
+if the number of active connections exceeds 4 times the number of cores in your cluster your application is likely not achieving maximum throughput, and you should reduce the maximum number of connections in your connection pool, or consider adding more cores to each node in your cluster.
```

Comment:
"or consider adding more cores to each node in your cluster." is maybe a bit too specific. Another option is to add more nodes to the cluster.

So this could be rephrased as "scale the cluster by adding more nodes or cores per node" or something.